### ISSUE TITLE
Sex Parameter for clinical use

### ISSUE THREAD

**Issue by kruzikh** [hl7-eu/laboratory#74]
I propose to remove Patient Sex Parameter For Clinical Use from the Patient resource and include it into Supporting information instead https://github.com/hl7-eu/laboratory/issues/73.

Rational: patient resource includes mostly demographic information. Sex parameter is a clinical information and should be present somewhere else, not in the patient resource.

**Comment by joofio** [comment]
Not sure if I follow. Removing the patient.gender (0..0?) from patient and add it in another section?

**Comment by kruzikh** [comment]
No, administrative gender will remain where it is. This is only about sex for clinical use element.

Best
Hynek

Odesláno z aplikace Outlook pro iOS<https://aka.ms/o0ukef>
________________________________
Od: Joao Almeida ***@***.***>
Odesláno: Wednesday, October 23, 2024 5:29:01 PM
Komu: hl7-eu/laboratory ***@***.***>
Kopie: Hynek ***@***.***>; Author ***@***.***>
Předmět: Re: [hl7-eu/laboratory] Sex Parameter for clinical use (Issue #74)


Not sure if I follow. Removing the patient.gender (0..0?) from patient and add it in another section?

**Comment by VeraNoPhD** [comment]
Hey Hynek! 

Would you propose to use Observation for sex at birth and reference it in supporting information? 

BR
Vera

**Comment by joofio** [comment]
ah the extension. There are already some existing extensions for these subjects: https://hl7.org/fhir/extensions/extensions-Patient.html
does any of them work?

**Comment by kruzikh** [comment]
It is a topic for discussion. We can present it as observation, however an extension that we already have used ([Patient Sex Parameter For Clinical Use](https://hl7.org/fhir/extensions/StructureDefinition-patient-sexParameterForClinicalUse.html)) is also an option. This extension seems to be  covering exactly our needs and can by applied to any resource.
So far, we used this extension to extend [EU patient](https://build.fhir.org/ig/hl7-eu/laboratory/StructureDefinition-Patient-eu-lab.html) resource. The question is if it is the right option, or if we should rather see it as a clinical information attached to the patient but provided somewhere else then in the patient resource itself. e.g. in the [serviceRequest.supportingInformation](https://build.fhir.org/ig/hl7-eu/laboratory/StructureDefinition-ServiceRequest-eu-lab.html).
